content/doc/book/installing/_master-agent-machines-setup-windows.adoc

```diff
@@ -0,0 +1,61 @@
+.. Select the appropriate *Usage* setting:
+... For an additional worker: _Utilize this node as much as possible_
+... For specialized jobs: _Leave this machine for tied jobs only_
+.. *Launch Method*:
+... An easy way to control a Windows agent is by using _Launch agent via Java Web Start_  (Recommended for Windows)
+... TODO: add steps for other methods.
+.. *Availability*
+... _Keep this agent online as much as possible_
+... TODO: add details for each option.
+.. Press *Save*. +
+image:/images/installing/setup06.png[Save Jenkins agent definition,width=80%]
+. Now you need to connect your agent machine to the master using the following steps.
+.. Open a browser on the **agent*** machine* and go to the *Jenkins master server* url \http://yourjenkinsmaster:8080/ .
+.. Go to *Manage Jenkins* > *Manage Nodes*,
+... Click on the newly created agent machine. You will need to login as someone that has the "Connect" Agent permission if you have configured global security.
+.. Click on the *Launch* button to launch agent from browser on agent. +
+image:/images/installing/setup03.png[Connect Jenkins agent,width=90%]
+.. Run the program. +
+image:/images/installing/setup04.png[Run the program] +
+If Windows asks you to choose a program, use `+$JAVA_HOME\bin\javaws.exe+` (the Java Web Start Launcher).
+.. If you encounter connection issue, then you could enlarge the popup windows to see the master *port used* and check your network configuration (firewall, port forward, ...).
+image:/images/installing/connect.jpg[Jenkins agent Windows dialog,width=70%] +
+Note that Jenkins chooses a random, high-number port. If you want Jenkins to use a static port, go to "Manage Jenkins" > "Configure Global Security" and choose the port number in the "TCP port for JNLP agents" box.
+.. If the port is open, the agent still can't connect, and your Jenkins instance is served securely over SSL/HTTPS, assure that you're running a recent Java version so that it includes the correct Java Cryptography Extension (JCE).
+.. Now you should see the agent machine connected under *Nodes*. +
+image:/images/installing/Node.png[Connected node, width=50%]
+
+.  If you want the service to run on start-up of the agent machine do the following (Windows only directions):
+.. In the agent program running on your agent machine,
+.. click *File* --> *Install as Windows Service. * +
+image:/images/installing/setup05.png[Install as Windows service dialog] +
+Note that this feature requires ".Net Framework"
+.. *Start*, type Services and Select the *Services* program.
+.. Find *Jenkins Agent* in the list, Double click to open.
+.. Select *Startup type* --> *Automatic*.
+.. Go to the *Log On* tab, change the *Log on as* to a user of your choice (Special user account Jenkins recommended).
+.. Make sure that auto login is set for the agent machine for the user account, then the VM (or physical computer) should connect and be available when needed.
+
+TODO: Add steps for connecting via ssh
```

Comment:
Need this TODO resolved (especially since ssh agents are much easier to manage than inbound connection agents).  Refer to the [ssh build agents configuration guide](https://github.com/jenkinsci/ssh-slaves-plugin/blob/master/doc/CONFIGURE.md#launch-windows-agents-using-microsoft-openssh) for details